ngraph/core/reference/include/ngraph/runtime/reference/ctc_loss.hpp

```diff
@@ -50,9 +50,9 @@ namespace ngraph
                             std::string("Logit or label length cannot greater than max sequence"
                                         "length. Also a label length cannot be greater than a"
                                         "logit length.\nMaxSeqLen: ") +
-                            std::to_string(maxTime) + "; Logit len: " +
-                            std::to_string(actualLogitLen) + "; Label len: " +
-                            std::to_string(actualTargetLen));
+                            std::to_string(maxTime) +
+                            "; Logit len: " + std::to_string(actualLogitLen) +
+                            "; Label len: " + std::to_string(actualTargetLen));
```

Comment:
the old one was better IMO

Reply:
I didn't find style configuration flag to align this with the old style.